sirepo/package_data/static/js/sirepo-lattice.js

```diff
@@ -1263,7 +1264,7 @@ SIREPO.app.directive('lattice', function(appState, latticeService, panelState, p
             const ABSOLUTE_POSITION_TYPE = 'absolutePosition';
             $scope.plotStyle = $scope.flatten ? '' : 'cursor: zoom-in;';
             $scope.isClientOnly = true;
-            $scope.margin = $scope.flatten ? 0 : 3;
+            $scope.margin = $scope.margin || ($scope.flatten ? 0 : 3);
```

Comment:
@moellep I don't love this. The reason we need to add margin is because labels were being placed outside of the viewable field when zoomed all of the way out of the beamline. Since the labels aren't part of the svg I'm not sure there is a better way to handle this.